src/styles.css

```diff
@@ -13,6 +13,15 @@ body {
     text-indent: 0ex;
 }
 
+.hr {
+    overflow: visible; /* For IE */
+    padding: 0;
+    border: none;
+    border-top: medium double #333;
```

Comment:
why double? do we want an `hr` to appear as two lines?

Reply:
I've no opinion on that... Whatever people prefer. The bug seems to be fixed moving from an `<hr>` to a `<div>` --- not really sure *why* and we have the same issue with the `<image/>` tag.